src/AddTodoForm.js

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
```

Comment:
When you're only importing a single default import, you can just say 
```suggestion
import React from 'react';
```

the `import * as x` is more used for when you want to import a whole bunch of imports together, which is a pretty obscure case that I don't think you'll ever need in this class.